docs/UserGuide.md

```diff
@@ -176,6 +176,20 @@ Examples:
 * `list` followed by `listOrder 2` lists all orders of the 2nd person in the address book.
 * `find Betsy` followed by `listOrder 1` lists all orders of the 1st person from the list of persons found with "Betsy".
 
+### Deleting an order under a Person : `deleteOrder`
+
+Deletes a specified order under a selected person from the address book.
+
+Format: `deleteOrder INDEX`
+
+* Deletes an order under the selected person at the specified `INDEX`.
```

Comment:
Shouldn't you include that a person's orders need to be listed using `listOrder` command first before you can delete an order?